frontend/src/app/layout.tsx

```diff
@@ -73,6 +74,7 @@ export default function RootLayout({
             <NotificationManager />
             <AppRouterCacheProvider>
               <ThemeProvider>
+                <FeatureCTABanner actionUrl="/signup" />
```

Comment:
could the banner be under the navbar?

Reply:
when I was poking around, it seems more common to be at the top now! here's a screen shot
<img width="1695" alt="Screenshot 2025-04-29 at 11 21 56 AM" src="https://github.com/user-attachments/assets/57c81191-186f-4b05-867f-35d99fe3bfd2" />